doc/report.tex

```diff
@@ -28,13 +28,11 @@
 
 \section{Overview}
 
-MYRC is a java-based distributed chat application which sought to implement
-a subset of the IRC protocol as faithfully as possible, with the exception that
-communication between the server and clients is done via sockets, rather
-than over TCP, as the protocol requires.
```

Comment:
Removed comment about difference between sockets and TCP because sockets are actually implemented over TCP.